extensions/bot-mode.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Bot Mode
+layout: spec
+meta-description: A user mode that indicates a user is a bot.
+copyrights:
+  -
+    name: "Daniel Oaks"
+    period: "2021"
+    email: "daniel@danieloaks.net"
+---
+
+## Introduction
+This is a standardised mode that lets clients mark themselves as bots, and lets other clients see them as bots.
+
+## The `BOT` ISUPPORT token
+Servers publishing the `BOT` [ISUPPORT](https://modern.ircdocs.horse/#feature-advertisement) token let clients mark themselves as bots by setting a user mode. The value of the `BOT` token is the mode character which is used to enable bot mode (e.g. `BOT=B` or `BOT=b`).
+
+When a client is marked as a bot, they are given a special numeric as part of their `WHOIS` response, it is indicated in their `WHO` flags, and servers may include the `bot` tag on that client's outgoing messages.
+
+## The `RPL_WHOISBOT` `(335)` numeric
+This numeric is returned as part of a bot's `WHOIS` reply.
+
+## The `B` `WHO` flag
+When a `RPL_WHOREPLY` `(352)` numeric is returned for a bot, the `"B"` character is returned in the flags (alongside `H|G`).
+
+## The `bot` tag
+The `bot` tag indicates that the given user is a bot. This tag SHOULD be added by the server to all commands sent by a bot (e.g. `PRIVMSG`, `JOIN`, `MODE`, `NOTICE`, and all others). The tag SHOULD also be added by the ircd to all numerics directly caused by the bot. This tag MUST only be sent to users who have requested the `message-tags` capability.
```

Comment:
`This tag MUST only be sent to users who have requested` -> `This tag MUST be sent to users if and only if they have requested`

Reply:
double negatives feel a bit hard to process?

Reply:
`This tag MUST NOT be sent to users who have not requested` also allows clients to not send the tag to clients that did request it.

Reply:
@MaxLeiter It's a S2C tag not a C2S tag.

Reply:
I'm gonna mark this as resolved for now, I prefer the original wording a bit more